Project2015To2017/Transforms/FileTransformation.cs

```diff
@@ -173,19 +172,61 @@ private static bool KeepFileInclusion(XElement x, Project project)
 				return false;
 			}
 
+			if (linkElement == null)
```

Comment:
Moved `Include` -> `Update` transformation to be **after** check if file is in project folder. Link is merely IDE UX guidance, it is not related to actual inclusion. If file is not in project folder, `Update` is not what we want..